ipywidgets_jsonschema/form.py

```diff
@@ -931,12 +931,19 @@ def _resetter():
         # Initially call the resetter
         _resetter()
 
+        def _getter():
+            result = [h.getter() for h in elements[:element_size]]
+            if schema.get("set", False):
```

Comment:
I think there is a mix-up here of `set` and `uniqueItems`. `uniqueItems` is deprecated in a pydantic field definition, but it is still very much in the generated JSONSchema. `set` on the other hand does not exist in the JSONSchema spec - but `uniqueItems` does.